newIDE/app/src/MainFrame/index.js

```diff
@@ -2353,7 +2409,11 @@ const MainFrame = (props: Props) => {
 
   const saveProjectAs = React.useCallback(
     () => {
-      if (!currentProject) return;
+      if (!currentProject || !!checkedOutVersionStatus) {
```

Comment:
Nitpicking: can you factor this with the boolean `canSaveProjectAs`? So that your useful comment can be used for the boolean.